cfi_backward.adoc

```diff
@@ -420,13 +420,13 @@ The following faults may occur:
 .. Instructions fetch causes a page fault
 . if the accessed page is not a shadow stack page
 .. `ssamoswap` and `sspush` cause a store/AMO access fault
-.. `sspop` causes a load access fault
+.. `sspop` and `ssamoswap` cause a load access fault
```

Comment:
ssamoswap as noted in line 422 causes a store/AMO access fault.

Reply:
Thanks for pointing that out.
Let me fix it.

Reply:
fixed it. please see new commit on same branch.